src/main/java/de/retest/web/RecheckWebOptions.java

```diff
@@ -36,22 +32,44 @@ public static RecheckWebOptionsBuilder builder() {
 		private AutocheckingCheckNamingStrategy checkNamingStrategy = new CounterCheckNamingStrategy();
 		private ScreenshotProvider screenshotProvider = null;
 
-		public RecheckWebOptionsBuilder
+		/**
+		 * @param checkNamingStrategy
+		 *            The {@link AutocheckingCheckNamingStrategy} to be used. Default is
+		 *            {@link CounterCheckNamingStrategy}.
+		 * @return self
+		 */
+		protected RecheckWebOptionsBuilder
 				checkNamingStrategy( final AutocheckingCheckNamingStrategy checkNamingStrategy ) {
 			this.checkNamingStrategy = checkNamingStrategy;
 			return this;
 		}
 
+		/**
+		 * @param screenshotProvider
+		 *            The {@link ScreenshotProvider} to be used. Default is determined via
+		 *            <code>retest.properties</code>.
```

Comment:
The default is&mdash;if nothing is specified in `retest.properties` and system properties&mdash;`ScreenshotProviders#DEFAULT`